08-VendoringDependenciesAsSubmodules.adoc

```diff
@@ -135,6 +135,7 @@ From the submodule addition output:
 * "Submodule directory (3)" shows the new directory named `submodule` that was created to store the contents of the new submodule repository. Note that you'd normally not call this `submodule` but we're just using this name for these examples.
 
 You have successfully added the `GitInPracticeReduxSubmodule` submodule to the `GitInPracticeRedux` repository.
+The `GitInPracticeRedux` repository might now be referred to as the "superproject", i.e. a Git repository containing a submodule.
```

Comment:
```suggestion
We will now refer to the `GitInPracticeRedux` as the "superproject" i.e. the Git repository containing the submodule.
```

Reply:
maybe "We will now refer to `GitInPracticeRedux` as" (without "the") ?

Reply:
Good call, adjusted my suggestion accordingly